reana-ui/src/pages/profile/components/Quota.js

```diff
@@ -55,9 +57,86 @@ export default function Quota() {
     );
   }
   return (
-    <Grid columns={2}>
-      {renderPieChart("CPU", quota.cpu)}
-      {renderPieChart("HDD", quota.disk)}
-    </Grid>
+    <>
+      <Grid columns={2}>
+        {renderPieChart("CPU", quota.cpu)}
+        {renderPieChart("Disk", quota.disk)}
+      </Grid>
+      <div className={styles["quota-info"]}>
+        {getQuotaNotifications(quota, config.adminEmail).map(
+          (notification, index) => {
+            const message = (
+              <span>
+                {notification.body} Please see:{" "}
+                <a
+                  href={REANA_QUOTAS_DOCS_URL}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {REANA_QUOTAS_DOCS_URL}
+                </a>
+              </span>
+            );
+            return (
+              <div key={index}>
+                <Notification
+                  icon={notification.icon}
+                  header={notification.header}
+                  message={message}
+                  error={notification.type === "critical"}
+                  warning={notification.type === "warning"}
+                  closable={false}
+                />
+              </div>
+            );
+          }
+        )}
+      </div>
+    </>
   );
 }
+
+function getQuotaNotificationInfo(quotaType, exceededLimit, adminEmail) {
+  if (quotaType === "disk" && exceededLimit)
+    return "Please delete unnecessary workflows or files.";
+  if (quotaType === "disk" && !exceededLimit)
+    return "Please consider liberating disk space by deleting some workflows or files.";
+  if (quotaType === "cpu" && exceededLimit)
+    return (
+      <span>
+        Please contact{" "}
+        {adminEmail ? (
+          <a href={`mailto:${adminEmail}`}>REANA administrators</a>
+        ) : (
+          "REANA administrators"
+        )}{" "}
+        for quota limit increase.
+      </span>
+    );
+}
+
+export function getQuotaNotifications(quota, adminEmail) {
+  let notifications = [];
+  for (const [key, value] of Object.entries(quota)) {
+    const { usage, limit, health } = value;
+    if (health !== "critical") continue;
+    const percentage = Math.round((usage?.raw / limit?.raw) * 100);
```

Comment:
Why do we need `?.` ?

Reply:
For safety, just in case something is wrong with `usage` and `limit` values. E.g. it's `undefined`

Reply:
What I was thinking is that if either `usage?.raw` or `limit?.raw` is undefined, then `percentage` becomes `NaN` but we use it anyway in the following part of code. I am not sure how we should handle this, given that this issue is probably present also in other parts of the code (e.g. [here](https://github.com/reanahub/reana-ui/blob/9d7b4bbcd54b12da901ed7e24a04f347ec6f354e/reana-ui/src/pages/profile/components/Quota.js#L28)). What do you think? Should we address this in another issue?